timestamp

```diff
@@ -0,0 +1 @@
+1442009919.13
```

Comment:
@Carreau I think its related to the D3, but not sure. Auberon made it so it reloads the data every hour I believe but not sure if it needs that file there